data/SpeakerPack/_template.md.j2

```diff
@@ -0,0 +1,174 @@
+
+Either way, **you need to let us know before 15 August** through Pretalx. If you [edit your accepted submission](https://pretalx.com/pycon-au-2020/me/submissions/), there will be a new question you can use to indicate your preference.
+
+{% endif %}
+
+## Technical requirements
+
+{% if type == 'live' and room == 'curlyboi' %}
+
+The stream you're on is getting the white-glove treatment! The AV team will get in touch with you beforehand to run through the requirements for your stream.
+
+{% elif type == 'live' and room == 'other' %}
+
+- You'll need a webcam and microphone. If you have a webcam built into your laptop it's probably fine, but it's a good idea to plug in an external microphone if at all possible.
+- You'll need the latest **Chrome** or **Firefox**. It's also a good idea to have **Skype** or **Zoom** installed, as we can fall back to these as a backup, but you don't have to.
+- You'll need a stable Internet connection. If you know you’ve got Internet issues (i.e. you usually have issues with video conferencing with work colleagues, your wifi drops out a lot, etc), please consider pre-recording instead.
+- You'll need a secondary monitor plugged into your computer, **or** a secondary device (such as a spare laptop, phone or tablet, but make sure it's set not to go to sleep!)
+
+{% elif type == 'pre-record' %}
+
+- Video length: **two minutes shorter** than the duration you selected&mdash;so either **13, 23 or 53 minutes**. It can run a little shorter but it **can not** run longer, even by a second.
+- Resolution: {% if room == 'curlyboi' %}1080p (1920x1080){% else %}720p (1280x720){% endif %}
+{%- if day == 'main' %}
+    - **Note**: If you don't know which stream you'll be in yet, note that the Curlyboi Theatre is 1080p, and the other streams are 720p.
+{%- endif %}
+- We can handle format re-conversion, but preference for format is below.
+- We will email out upload details to those who opt-in to this.
+- No video intro or outro sequence needed&mdash;if you include one, it may be edited out.
+- You must be available in the conference Venueless stage during your talk, so you can answer questions!
+- Editing needs to be done by you, and requesting changes after the due date is not possible. The AV team is expecting to have many hours of video to work through so we likely will not have the capacity to do special pre-event edits ourselves.
+{%- if room == 'curlyboi' %}
+- The AV team will be in touch to help out, as we're expecting to help edit some videos for this track specifically.
+{%- endif %}
+
+{% endif %}
+
+Please also look through our AV team's **quality tips**. They're really important reading, so we've made the button to go read them very big and very green.
+
+<div class='center-content'>
+<a href="https://forms.gle/F6JhtbFoZhVBzvz16" class='btn btn-chonk'>I want to look (and sound) good!</a>
```

Comment:
this link is wrong

Reply:
Yep, it's because that document hasn't been put anywhere publicly accessible yet! It's coming :)